nodejs/eks/nodegroup.ts

```diff
@@ -677,8 +701,48 @@ export type ManagedNodeGroupOptions = Omit<aws.eks.NodeGroupArgs, "clusterName"
     scalingConfig?: pulumi.Input<awsInputs.eks.NodeGroupScalingConfig>
 };
 
-export function createManagedNodeGroup(name: string, args: ManagedNodeGroupOptions, parent: pulumi.ComponentResource): aws.eks.NodeGroup {
+export function createManagedNodeGroup(name: string, args: ManagedNodeGroupOptions): aws.eks.NodeGroup {
```

Comment:
Would it make sense to make the `parent` an optional param and ignore it to avoid changing the signature in existing programs? You could print a warning that it's not used.

Reply:
Done. parent is now optional and defaulted.